gluoncv/model_zoo/monodepthv2/monodepth2.py

```diff
@@ -105,12 +105,13 @@ def get_monodepth2(backbone='resnet18', pretrained_base=True, num_input_images=1
     """
     acronyms = {
         'kitti_stereo_640x192': 'kitti_stereo_640x192',
+        'kitti_mono_640x192': 'kitti_mono_640x192',
```

Comment:
I didn't see any difference between the key and value here. I think they are the same string. May I ask why do we need this `acronyms ` dictionary?

Reply:
I refer to the code of segmentation model zoo here, but acronyms are not needed actually. Modified, thanks for reminding!